### DISCUSSION TITLE
QQ, what are the criteria for similar notes?

### DISCUSSION THREAD

**Discussion by richardap1** [TriliumNext/Trilium#3631] [Q&A] [answered]
As a general newbie to Trilium I was clicking around and couldn't figure out how similar notes are connected.   For example, clicking on a work note brought you about 50 recipes that I had imported from Evernote.

**Comment by zadam** [comment, marked as answer]
There's several factors - mostly textual similarity of the titles and attributes. It's not very predictable, and this feature has generally not proved to be very useful.
